src/components/tiles/TilesList.tsx

```diff
@@ -29,7 +30,7 @@ export function TilesList({ navigation, styles: instanceStyles }: Props) {
   const todayTileEnabled = data?.homeTab?.tiles?.includes?.('todayTile');
   const myDataTileEnabled = data?.homeTab?.tiles?.includes?.('myDataTile');
   const todayTile = data?.homeTab?.todayTile;
-
+  const messagesTileEnabled = data?.homeTab?.tiles?.includes?.('messagesTile');
```

Comment:
This is the equivalent of feature flagging this off, merging this will not impact anyone.

Reply:
This will need to be:
```
const messagesTileEnabled = data?.homeTab?.messageTiles?.length
```

See:
https://github.com/lifeomic/life-research-service/pull/116

Reply:
👍 I wasn't worrying about the specifics in this PR but wanted a placeholder here for when we worked those all out. I will certainly update here shortly.

Reply:
Updated but still hanging on until I know what the final API shakes out to be.